domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Income.java

```diff
@@ -12,6 +13,8 @@
 
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
+// Create these fields in JSON when serialize Java object, ignore them when deserialize.
```

Comment:
pointless having comment in one place and not the others, I recommend removal
REQ